test/chunking-form/samples/aliasing-extensions/_expected/amd/main1.js

```diff
@@ -1,7 +1,7 @@
 define(['require'], function (require) { 'use strict';
 
 	console.log('main1');
-	new Promise(function (resolve, reject) { require(['./generated-chunk.js'], resolve, reject) });
-	new Promise(function (resolve, reject) { require(['./generated-chunk2.js'], resolve, reject) });
+	new Promise(function (resolve, reject) { require([('./generated-chunk.js')], resolve, reject) });
```

Comment:
Great work! These added parentheses are basically the only thing that is not as nice as it could be. But there is a possible solution...